crates/hotshot/src/traits/networking/web_server_network.rs

```diff
@@ -192,9 +194,9 @@ struct Inner<TYPES: NodeType> {
     /// Our own key
     _own_key: TYPES::SignatureKey,
     /// Queue for broadcasted messages
-    broadcast_poll_queue: Arc<RwLock<Vec<RecvMsg<Message<TYPES>>>>>,
+    broadcast_poll_queue_0_1: Arc<RwLock<Vec<RecvMsg<Message<TYPES>>>>>,
     /// Queue for direct messages
-    direct_poll_queue: Arc<RwLock<Vec<RecvMsg<Message<TYPES>>>>>,
+    direct_poll_queue_0_1: Arc<RwLock<Vec<RecvMsg<Message<TYPES>>>>>,
```

Comment:
If we are going to use `Inner` for all versions, I think these should have names without `_0_1`. Otherwise, the struct should be `Inner_0_1`. If we need a way to distinguish version numbers, we could maybe add `Version` as a generic type.

Reply:
My thought here was that we should separate messages into different queues depending on the version number, but `Inner` would contain shared state needed between the different version numbers.

But I think I also don't fully understand where e.g. `broadcast_poll_queue` is consumed. I'll bring this up during pair programming tomorrow!

Reply:
As discussed in the last pair programming, this makes sense!